### ISSUE TITLE
Tizen on macOS

### ISSUE THREAD

**Issue by amirvenus** [xamarin/Xamarin.Forms#2907]
Hi guys,
Has anyone had any success running Xamarin.Forms.Tizen on macOS?

I read this article:
https://blog.xamarin.com/add-tizen-projects-xamarin-forms-apps/

It sounds really promising and it mentions that I need an addin for VS Mac but I am really loat with lack of documentation.

**Comment by rookiejava** [comment]
If you wanna run your Xamarin.Forms.Tizen app on macOS, you should install [Tizen Studio](https://developer.tizen.org/development/tizen-studio/download) in advance.  

And, with latest version of Visual stduio for mac (7.5 or above), any addin for tizen project is no longer required for recognizing the tizen proejct. However, unfortunately, Visual studio for mac is not officially support to Tizen project. So, you have to get Tizen project which is created by using Windows visual studio or just modify the TargetFramework to `tizen40` in project file (`*.csproj`). After building/packaging the tizen project, you can see the application package (*.tpk). Finally, you can just install the *.tpk to your tizen emulator or device. How to install?
 a) drag and drop : Open the Finder and find and drag the *.tpk and drop to the emulator
 b) use sdb : similar to android's `adb`, tizen provides `sdb`.  `#sdb install *.tpk`

In short,
1. Install the Tizen studio and run emulator manager (refer to [guide](https://developer.tizen.org/development/tizen-studio))
2. With the latest Visual studio for mac, add/open the Tizen proejct to your solution. Make sure that the target framework should be `tizen40` and `tizen-manifest.xml` file should be required for packaging. (please refer to [sample apps PR ](https://github.com/xamarin/xamarin-forms-samples/pull/245))
3. Install the application to the emulator/device by using sdb or drag and drop.

Enjoy it.
